tests/unit/backend/corpora/common/test_authorizer.py

```diff
@@ -1,23 +1,27 @@
+import json
 import os
 import unittest
 
 import requests
 from chalice import UnauthorizedError
 
 from backend.corpora.common.authorizer import assert_authorized
+from backend.corpora.common.utils.aws_secret import AwsSecret
 
 if not os.getenv("DEPLOYMENT_STAGE"):  # noqa
     os.environ["DEPLOYMENT_STAGE"] = "test"  # noqa
 
+secret_name = "corpora/cicd/dev/auth0-secret"
```

Comment:
I think the correct paradigm here would be to set it as a class variable in a setUp function instead of as a global variable.

Reply:
moved to class variable